graphql/resolvers.go

```diff
@@ -536,6 +539,16 @@ func (r *queryResolver) User(ctx context.Context) (*restModel.APIUser, error) {
 	return &user, nil
 }
 
+type taskResolver struct{ *Resolver }
+
+func (r *taskResolver) PatchNumber(ctx context.Context, obj *restModel.APITask) (*int, error) {
+	patch, err := r.sc.FindPatchById(*obj.Version)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error retrieving patch %s: %s", *obj.Version, err.Error()))
+	}
```

Comment:
it could be a good practice to send an error if patch is nil

Reply:
In general yes, this function specifically won't ever return nil,nil so it's not needed here (but feel free to add it if you want)